testing/src/cypress/page-objects/Institution-objects/InstitutionProfileObject.ts

```diff
@@ -88,18 +88,22 @@ export default class InstitutionProfileObject {
   }
 
   countryInputText() {
-    return cy.get(".form-control").eq(16);
+    return cy.xpath("//div[@id='e96525o']//div[@role='combobox']");
```

Comment:
Just double checking, what is the reason why `data-cy` is not used here?
Is there a technical issue or the below is not the right one?
![image](https://user-images.githubusercontent.com/61259237/173408143-30bf61f4-4249-4950-8c8a-6844e8181ad3.png)

Reply:
Why is that? Can you elaborate more, please? The `data-cy` is right on the html `select`, right?

Reply:
Reason for "Why data-cy is not possible here in dropdown".

When data-cy is used in a "country" dropdown, elements are not highlighted because data-cy is not added at the proper place to identify those dropdown values.
![note-proper](https://user-images.githubusercontent.com/98920571/173484152-0a2a4b96-91ba-40af-a7f2-eedcfc7b47f1.jpg)


Data-cy is not getting used here, so I am using xpath instead, & you can see that the dropdown is getting highlighted when using xpath selector.
<img width="955" alt="image" src="https://user-images.githubusercontent.com/98920571/173484812-11b3505a-147c-435c-b324-4fdedbff94f1.png">


Example for where data-cy is used for - In textbox elements, data-cy identifies and highlighted elements when searching with its selector (data-cy).
![proper-data-cy](https://user-images.githubusercontent.com/98920571/173484534-19c4338d-211a-420b-b63d-81d46c65446a.jpg)

Reply:
> Folders looks better. Just left minor comments, please take a look.

Replied. Thanks for quick review.

Reply:
As we talked it seems that it is technically possible. I will resolve this assuming that the upcoming PR will explore a better solution.